include/boost/geometry/algorithms/detail/distance/geometry_to_segment_or_box.hpp

```diff
@@ -239,7 +266,7 @@ class geometry_to_segment_or_box
 
         // consider all distances of the points in the geometry to the
         // segment or box
-        comparable_return_type cd_min1(0);
+        comparable_return_type cd_min1;
```

Comment:
Do I understand correctly that this is because `comparable_return_type` is no longer a number? If that's the case maybe it would be wise to rename it because in the future we may get an impression that it is comparable distance. What do you think?

Reply:
Since the name is comparable return type and not comparable distance it does not yield a problem for me. But I changed the name of the variables containing `cd` to `cr`.